docs/hawkey/CMakeLists.txt

```diff
@@ -1,22 +1,24 @@
 # html and man documentation are separate targets, apparently there's no way to
 # tell sphinx-build to do them both in one go:
 
-if (${PYTHON_VERSION_MAJOR} STREQUAL "2")
-    SET(SPHINX_BUILD_NAME "sphinx-build")
+if(${PYTHON_VERSION_MAJOR} STREQUAL "2")
+    set(SPHINX_BUILD_NAME "sphinx-build")
 else()
-    SET(SPHINX_BUILD_NAME "sphinx-build-3")
+    set(SPHINX_BUILD_NAME "sphinx-build-3")
```

Comment:
Could we just use `sphinx-build-PYTHON_VERSION` here? We should be able to interrogate that information from the main CMakeLists.

Reply:
Unfortunately not. There's no sphinx-build-2 on RHEL 7 :(